screens/Home.js

```diff
@@ -105,6 +104,10 @@ class Home extends React.Component {
       Strings.setLanguage(this.props.language);
     }
 
+    if(!this.props.fontFace || this.props.fontFace === "GurbaniAkharSG"){
+      this.props.setFontFace("GurbaniAkharTrue");
+    }
+
```

Comment:
Do you want to check if the `this.props.fontFace` is `GurbaniAkharSG` only then the font should be set to `GurbaniAkharTrue`?

Reply:
Fixed

Reply:
Got it veerji